### DISCUSSION TITLE
Build folder not created after npm run build

### DISCUSSION THREAD

**Discussion by jelluh24** [photoview/photoview#712] [General]
I'm trying to install Photoview without docker on my machine, but I'm running into the following problem:
 I have installed node.js 16.16.0 and npm (which it didn't install automatically, had some trouble with installing this new version of node as well). Then when I run npm install and npm run build in the /opt/photoview/ui directory it seems to go well, but then there's supposed to be a build folder in that same directory, but there isn't. I tried making the folder myself but that doesn't seem to work either. I even modified the package.json file to install it to a custom directory but no output there either. How do I fix this?

**Comment by viktorstrate** [comment]
It's recently been renamed to `dist`, try and look for that instead
